vllm/envs.py

```diff
@@ -1204,6 +1205,10 @@ def get_vllm_port() -> Optional[int]:
     # Add optional custom scopes for profiling, disable to avoid overheads
     "VLLM_CUSTOM_SCOPES_FOR_PROFILING":
     lambda: bool(int(os.getenv("VLLM_CUSTOM_SCOPES_FOR_PROFILING", "0"))),
+
+    # If set to 1, use UCC allreduce
+    "VLLM_USE_UCC_ALLREDUCE":
```

Comment:
Will UCC only benefit for all-reduce? say, how about other primitives like all-2-all?

Reply:
The current PR scope is only enabling `all_reduce` for tensor parallel scenario. (Others collectives might be added later.)

Reply:
make sense